FogCreek/Glitch-Community
# Storm scene: miscellaneous popover componentizations

## Links
* Remix link https://storm-scene.glitch.me/

## Changes:
* componentize Delete Project popover
* use Popover components on search form & homepage video
* remove unused (?) sign-in page?

## How To Test:
* visit https://storm-scene.glitch.me/
* verify that the "how it works" video opens and closes if it loses focus
* verify that the search autocomplete opens when you type in the search box and closes when you focus elsewhere
* log in
* create a project
* delete that project from its project page
* verify that the delete project pop works
